### ISSUE TITLE
‘Cannot create __weak reference in file using manual reference counting’

### ISSUE THREAD

**Issue by qiubaiying** [ReactiveCocoa/ReactiveObjC#126]
Pod load ReactiveObjC compile errors：

`Cannot create __weak reference in file using manual reference counting`

To solve：

Set `Weak References in Manual Retain Release:YES`

![](https://ws4.sinaimg.cn/large/006tKfTcgy1fl3rxtkgtwj30qb0axgnn.jpg)

**Comment by G1King** [comment]
yes,I have also encountered this problem, I am introducing class (MRC).

**Comment by qiubaiying** [comment]
Ha ha, I encounter this problem again in the new project, and then found the issue was previously created.
